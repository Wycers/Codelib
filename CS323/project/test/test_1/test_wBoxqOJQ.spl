int test_1_r02(int a, int b)
{
  if(){}
  if (a > b)
  {
    return 1_a;
  }
}