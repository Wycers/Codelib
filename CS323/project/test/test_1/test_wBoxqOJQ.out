Error type B at Line 3: Missing exp when if
Error type A at Line 6: unknown lexeme 1_a
